Tests/integration_tests/test_e2e_demisto_sdk_flow_foo.py

```diff
@@ -0,0 +1,46 @@
+import random
+
+from e2e_tests_utils import e2e_tests_utils
+
+
+def test_e2e_demisto_sdk_flow_foo(tmpdir, insecure: bool = False):
+    # Importing TestSuite classes from Demisto-SDK
+    e2e_tests_utils.cli(f'mkdir {tmpdir}/git')
```

Comment:
doesn't `tmpdir` create the dir for us? do we really need to `mkdir`?

Reply:
I use `tmpdir` for several things:

- The sdk repo clone destination
- The pack I created for this test

Reply:
I use tmpdir for several things:

- The sdk repo clone destination
- The pack I created for this test